lib/src/base-styles/_nx-tile.scss

```diff
@@ -185,10 +206,10 @@ $nx-tile-background-color: $nx-white;
 .nx-tile-footer {
   @include container-horizontal;
 
-  border: 0 solid $nx-grey-4;
-  border-top-width: 1px;
-  margin-top: 20px;
-  padding: 12px 4px 0 4px;
+  border-top: 1px solid #e9e9e9;
+  margin-top: $nx-spacing-xl;
+  padding: $nx-spacing-xl 0;
+  text-align: right;
```

Comment:
Is this always the case?  Is the footer always going to be just buttons?  Why are these button to the right when footer buttons everywhere else are to the left (or is that changing globally)?  If it is changing globally, maybe we want to put this style on `.nx-btn-bar` instead of here.

Reply:
> Is this always the case? Is the footer always going to be just buttons? Why are these button to the right when footer buttons everywhere else are to the left (or is that changing globally)? If it is changing globally, maybe we want to put this style on `.nx-btn-bar` instead of here.

As far as I understand it's global @dturner32 ?

We would want to add it to `.nx-btn-bar` for consistency but we don't nest `.nx-btn-bar` inside of footers so I don't see the connection WRT to this PR. `.nx-btn-bar` is for when there isn't a container like a footer to hold the buttons.

Reply:
Yes, the buttons are being moved to the right as a global change. This helps with visual flow as a user is reading from left to right. Their eye more naturally falls on the button placed in the bottom right.